File: apps/web/fix_syntax.js
const fs = require('fs');
const path = 'e:\\localweb\\Local-Business-Listing-Directctory\\apps\\web\\app\\business\\[slug]\\BusinessDetailClient.tsx';
let content = fs.readFileSync(path, 'utf8');

// The extra braces are at line 739.
// I'll look for "})}}" and replace with "})}"
const wrongBraces = '})}}';
const correctBraces = '})}';

if (content.includes(wrongBraces)) {
    content = content.replace(wrongBraces, correctBraces);
    fs.writeFileSync(path, content, 'utf8');
    console.log('Successfully fixed extra braces in BusinessDetailClient.tsx');
} else {
    console.error('Could not find sequence "})}}"');
}

File: apps/web/brain/7204f3d1-a072-4086-815e-23c745d215e7/scratch/fix_syntax.py
import os

file_path = r"e:\localweb\Local-Business-Listing-directory\apps\web\app\(dashboard)\settings\page.tsx"
with open(file_path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

for i in range(len(lines)):
    if '                            }}>' in lines[i]:
        lines[i] = lines[i].replace('                            }}>', '                            `}>')
        print(f"Replaced line {i+1}")

with open(file_path, 'w', encoding='utf-8') as f:
    f.writelines(lines)
